src/init.cpp

```diff
@@ -366,7 +366,7 @@ void SetupServerArgs()
     gArgs.AddArg("-blocknotify=<cmd>", "Execute command when the best block changes (%s in cmd is replaced by block hash)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
 #endif
     gArgs.AddArg("-blockreconstructionextratxn=<n>", strprintf("Extra transactions to keep in memory for compact block reconstructions (default: %u)", DEFAULT_BLOCK_RECONSTRUCTION_EXTRA_TXN), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
-    gArgs.AddArg("-blocksonly", strprintf("Whether to reject transactions from network peers. Transactions from the wallet, RPC and relay whitelisted inbound peers are not affected. (default: %u)", DEFAULT_BLOCKSONLY), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-blocksonly", strprintf("Whether to reject transactions from network peers. Transactions from the wallet, RPC and relay whitelisted inbound peers are affected. (default: %u)", DEFAULT_BLOCKSONLY), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
This is not true. It depends on the parameter interaction. I'd rather describe that here.

Also you'd need to update doc/low_bandwith.md or whatever it is called

Reply:
Hey @MarcoFalke, I am sorry for my mistake, and thanks for your comments. Should I put it as the following?
> Whether to reject transactions from network peers. Transactions from the wallet, RPC and relay whitelisted inbound peers might be affected depending on the parameter interaction.

Reply:
@MarcoFalke I have made an update which includes your remarks in a333b82. Please, let me know if it is correct?